### PR TITLE
Prepare for release v0.3.0

### PR DESCRIPTION
**Pull request by tiyash-basu-frequenz**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 requires = [
   "setuptools == 67.7.2",
   "setuptools_scm[toml] == 7.1.0",
-  "frequenz-repo-config[api] == 0.2.0",
+  "frequenz-repo-config[api] == 0.3.0",
```
Comment (tiyash-basu-frequenz) [inline pyproject.toml]: Is this change required before making new releases?

Comment (tiyash-basu-frequenz) [reply pyproject.toml]: Okay, now that the CI run has finished, I can see that some of the tests are failing. So this probably needs to be removed.

But this is interesting. When we tag v0.3.0, this will still point to v0.2.0. Will this cause a problem?

Comment (tiyash-basu-frequenz) [reply pyproject.toml]: It is `frequenz-repo-config`, and not `frequenz-repo-common` :facepalm: . Sorry for the confusion.
Removing this line.

Comment (tiyash-basu-frequenz) [reply pyproject.toml]: Removed.

**Review by stefan-brus-frequenz — APPROVED** [review]
